garrysmod/lua/includes/util.lua

```diff
@@ -73,6 +73,7 @@ function PrintTable( t, indent, done )
 	for i = 1, #keys do
 		local key = keys[ i ]
 		local value = t[ key ]
+		key = (type(key) == "number") && "[" .. key .. "]" || "[\"" .. key .. "\"]"

```

Comment:
I think `isnumber` is faster than `type == "number"`?

Reply:
Also, I don't really see the need for brackets around numeric keys.

Reply:
isnt `isnumber` just a shortcut for `type == "number"` (which basicly means a function call more somewhere)?

with the brackets i think it would look more consistent when both have them (maybe they can even be removed on both but that way it looks more like a table) ^^

Reply:
I don't think it is internally.

The point of adding the brackets around the strings is so that they're not consistent with each other though.

Reply:
i mean it just looks nicer to have
```
["string"] = {
 ...
}
[2] = "number"
```

than 
```
["string"] = {
 ...
}
2 = "number"
```

or
```
"string" = {
 ...
}
2 = "number"
```

so for this i may wait for a maintainer to decide which one would suite more for gmod

Reply:
OK nevermind, I agree with you.